tests/test_run_state_manager_async.py

```
@@ -0,0 +1,63 @@
+import asyncio
+import time
+import pytest
+from pathlib import Path
+from swarm.api.services.run_state import RunStateManager
+
+def test_list_runs_async_non_blocking(tmp_path):
+    async def run_test():
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `run_test`

(ANN202)